src/app/helpers/app-initializer/app-initializer.ts

```diff
@@ -38,13 +38,11 @@ export class AppInitializer {
  * App values
  */
 export interface Values {
-  keys: {
-    googleMaps: string;
-  };
   brand: {
     name: string;
     title: string;
   };
+  links: Record<string, string>;
```

Comment:
I don't like that this is obscured/dynamic.

This interface should be a static specification for a config file.